src/openforms/js/components/admin/submissions/filter.js

```diff
@@ -0,0 +1,44 @@
+window.onload = function () {
+  // Retrieve the filter link that displays only triggered rules
+  triggeredLink = document.querySelector('.logic-rules-filter > .selected > a');
```

Comment:
we generally use strict javascript, which implies declaring all variables using `const` (preferably) or `let` (if the value needs to be re-assigned).

Global variables are a Bad Practice so we avoid using those.

Also. rather than `window.onload`, we tap into the `DOMContentLoaded` event, but even that may not be needed, e.g. in `js/components/admin/form-category.js`:

```js
if (document.readyState !== 'loading') {
  init();
} else {
  document.addEventListener('DOMContentLoaded', init);
}
```